### PR TITLE
Add suffixes to typos from "v" to "z"

### PR DESCRIPTION
**Pull request by int-y1**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -55805,13 +55881,20 @@ wavelnes->wavelines
...
 weahter->weather
 weahters->weathers
 weant->want, wean,
 weaponaries->weaponry, weaponries,
 weaponary->weaponry
 weappon->weapon
 weappons->weapons
+wearn->warn, wear, earn, learn,
+wearned->warned, earned, learned,
+wearning->warning, wearing, earning, learning,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:55895]: Also `yearning` and `wearying` (tiring)?

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -56160,6 +56252,8 @@ wnidow->window, widow,
 wnidows->windows, widows,
 woh->who
 wohle->whole
+woild->would, world, wild,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:56255]: Add `wold`?

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -56404,8 +56501,14 @@ wron->wrong
 wronf->wrong
 wronly->wrongly
 wront->wrong
+wrose->worse, wrote, whose,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:56504]: `arose`, `brose`, `erose`, `prose`?

File: codespell_lib/data/dictionary_rare.txt [hunk 4]
```diff
@@ -276,4 +276,5 @@ wither->either, whether, weather,
 withing->within
 wont->won't
 wounder->wonder
+wroth->worth, wrote,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary_rare.txt:279]: Also `wrath`?

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -54888,11 +54912,18 @@ valule->value
 valuled->valued
 valules->values
 valuling->valuing
+valume->volume, value,
```
[the remark this answers is not in the document]
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:54915]: I couldn't find this typo.

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -54888,11 +54912,18 @@ valule->value
...
+valumes->volumes, values,
 valus->values, value, talus,
 valuse->values, value,
+valuses->values
```
[the remark this answers is not in the document]
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:54919]: I'll add `valises`. I couldn't find the other typos.

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -55272,6 +55322,11 @@ vetran->veteran
 vetrans->veterans
 vetween->between
 vew->view
+vewed->viewed
```
[the remark this answers is not in the document]
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:55325]: I'll add `vowed` and `vexed`. I couldn't find the other typos.

File: codespell_lib/data/dictionary.txt [hunk 8]
```diff
@@ -55805,13 +55881,20 @@ wavelnes->wavelines
...
+wbsite->website
+wbsites->websites
 weahter->weather
 weahters->weathers
 weant->want, wean,
 weaponaries->weaponry, weaponries,
 weaponary->weaponry
 weappon->weapon
 weappons->weapons
+wearn->warn, wear, earn, learn,
```
[the remark this answers is not in the document]
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:55893]: I'll add `worn` and `yearn`. I couldn't find `wean`.

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:56255]: I couldn't find this typo.

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:56504]: I couldn't find these typos.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Comment by DimitriPapadopoulos** [comment]
@int-y1 Thank you for patience and persistence throughout this round of changes. Your hard work has been greatly appreciated.

**Pull request merged**